manke/rook-ceph/README.md

```diff
@@ -0,0 +1,38 @@
+rook-ceph
+=========
+
+how to access ceph toolbox pod
+---
+
+```bash
+kubectl -n rook-ceph exec -it $(kubectl -n rook-ceph get pod -l "app=rook-ceph-tools" -o jsonpath='{.items[0].metadata.name}') -- bash
+```
+
+useful toolbox commands:
+
+* `ceph status`
+* `ceph osd status`
+
+simple test deployment to verify that ceph backed PVs can be created
+---
+
+```bash
+kubectl apply -f ./demo/ceph-demo-pvc.yaml
+```
+
+complete ceph cluster tear down (will loose all data)
+---
+
+Run from rke user env:
+
+_Must be run prior to the node cleanup._
+
+```bash
+./scripts/rook-ceph-k8s-cleanup.sh
```

Comment:
This is no longer true because of `rook-remover`.  We should start cleaning up or remove the rook-ceph READMEs.

Reply:
yeah, you are right, i will remove de READMEs for now, probably will have to update them in the future

Reply:
It is possible for incorrect docs to actually be a worse scenario than have no docs at all.